src/components/play/PlayMultipleChoices.tsx

```diff
@@ -89,7 +92,25 @@ const PlayMultipleChoices = ({
             fullWidth
             {...computeStyles(choice, idx)}
           >
-            {choice.value}
+            {showCorrection &&
+            choice.explanation &&
+            response.choices?.contains(choice.value) ? (
+              <Grid container direction="column">
+                <Grid item>
+                  <Typography variant="body1">{choice.value}</Typography>
+                </Grid>
+                <Grid item>
+                  <Typography
+                    variant="caption"
+                    data-cy={buildMultipleChoiceExplanationPlayCy(idx)}
+                  >
+                    {choice.explanation}
+                  </Typography>
+                </Grid>
+              </Grid>
```

Comment:
If I'm not mistaken you don't need a Grid if you have a column direction? Not sure.

Reply:
Oh, I assume so actually. But I just needed a parent element for both texts. Would there be a better suggestion?